<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDLForm</class>
 <widget class="QMainWindow" name="MDLForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="OutputDisplay">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This is where the output from the MDL processes is displayed.</string>
    </property>
    <property name="whatsThis">
     <string>This is where the output from the MDL processes is displayed.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectInputButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>3D volume dataset of any type readable by ITK, or .raw with dimensions indicated by the filename (x*y*z) </string>
    </property>
    <property name="whatsThis">
     <string>3D volume dataset of any type readable by ITK, or .raw with dimensions indicated by the filename (x*y*z) </string>
    </property>
    <property name="text">
     <string>Select input image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectBackboneButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Where to store the backbones that result from segmentation.</string>
    </property>
    <property name="whatsThis">
     <string>Where to store the backbones that result from segmentation.</string>
    </property>
    <property name="text">
     <string>Select backbone output file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ComponentsSizeInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Used to remove objects smaller than this volume size.  Use higher values forlarger noisy objects.  Typical values range between 10 and 100.</string>
    </property>
    <property name="whatsThis">
     <string>Used to remove objects smaller than this volume size.  Use higher values forlarger noisy objects.</string>
    </property>
    <property name="inputMask">
     <string>900; </string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="ComponentsSizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected components size</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="OutputImageLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RunSkeletonizationButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Execute the skeletonization process.  This button is disabled until all required parameters have been provided.</string>
    </property>
    <property name="statusTip">
     <string>Execute the skeletonization process.  This button is disabled until all required parameters have been provided.</string>
    </property>
    <property name="whatsThis">
     <string>Execute the skeletonization process.  This button is disabled until all required parameters have been provided.</string>
    </property>
    <property name="text">
     <string>Run MDL skeletonization</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DeleteIntermediaryFilesCheckBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Should we delete intermediary files generated during segmentation?</string>
    </property>
    <property name="whatsThis">
     <string>Should we delete intermediary files generated during segmentation?</string>
    </property>
    <property name="text">
     <string>Delete intermediary files</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectSpinesButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select spines output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpinesLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
